app/src/main/res/layout/activity_child_registration.xml

```diff
@@ -8,54 +8,55 @@
     android:background="@drawable/upperbackgroundlb">
 
```

Comment:
can we do something about the way it looks? 

i don't care if we have to switch things out it just looks... sloppy still. like everything is really spread out and nothing sticks out. 

reference the other amazing me material for some inspiration.